testdata/test_failed_dbt/models/staging/staging_test02.sql

```diff
@@ -2,6 +2,6 @@
 
 
 SELECT
-    1 AS x,
+  1 AS x,
   2 AS y,
```

Comment:
Change to test again
```suggestion
    2 AS y,
```